test/index.test.js

```diff
@@ -1610,4 +1617,79 @@ describe('index', function () {
             );
         });
     });
+
+    describe('getScmContexts', () => {
+        it('returns a default scmContext', () => (
+            scm.getScmContexts().then((result) => {
```

Comment:
We changed the type of return value of `scm.getScmContexts()` from promise to array, so you need to fix this test.